src/main/scala/org/hammerlab/guacamole/reads/Read.scala

```diff
@@ -170,6 +169,7 @@ object Read extends Logging {
   def fromSAMRecordOpt(record: SAMRecord,
```

Comment:
The docstring should mention that behavior then. In general our [style guide][1] says to avoid abbrevs in method names.

[1]: https://hammerlab.atlassian.net/wiki/display/MSL/Software+Development#SoftwareDevelopment-General